app/services/brapi/v1/studies_queries.rb

```diff
@@ -0,0 +1,289 @@
+      where_query += get_posgresql_where_condition("plant_trials.id", id, where_atts_count)
+      where_atts << get_posgresql_att(id)  
+    end
+    
+    
+    # Until ORCID implementation is done, we only must retrieve published or not owned datasets
+    where_atts_count+= 1
+    where_query = where_query + (where_atts_count>1?" and ":" where ") 
+    where_query = where_query + " (plant_accessions.user_id IS NULL 
+      OR plant_accessions.published = TRUE) "
+    
+    # select clauses
+    # studyDbId, studyName, studyType, seasons, trialDbId, trialName, startDate, endDate, active 
+    # Here the location has a different meaning than studies-search.Location here refers more to a place than the country.
+    # location: locationDbId, name, abbreviation, countryCode, countryName, latitude, longitude, altitude, additionalInfo
+    select_query = get_select_distinct_clause(" plant_trials.id ")
+    select_query += <<-SQL.strip_heredoc
+      
+      plant_trials.user_id,
+      plant_trials.id as "studyDbId", 
+      plant_trials.plant_trial_name as "studyName", 
+      plant_trials.trial_year as "seasons",           
+      plant_trials.id as "trialDbId", 
+      plant_trials.plant_trial_name as "trialName", 
+      -- location 
+      countries.id as "locationDbId",
+      plant_trials.place_name as "name",
+      countries.country_code as "countryCode",
+      countries.country_name as "countryName",
+      plant_trials.latitude as "latitute",
+      plant_trials.longitude as "longitude",
+      plant_trials.altitude as "altitude",
+      plant_trials.terrain as "terrain",
+      plant_trials.soil_type as "soil_type",
+      -- contacts
+      plant_trials.contact_person as "email",
+      plant_trials.entered_by_whom as "entered_by_whom_email",
+      -- additional info
+      plant_trials.plant_trial_description as "studyDescription",
+      plant_trials.data_provenance as "dataProvenance",
+      plant_trials.data_owned_by as "dataOwnedBy"    
+      
+    SQL
+    
+    # TODO: "studyType": "Trial" -> NOT PRESENT IN OUR DB
+    # TODO: "seasons": ["2007 Spring", "2008 Fall"] -> seasons themselves not present in our DB
+    # TODO: "startDate": "2007-06-01" -> NOT PRESENT IN OUR DB
+    # TODO: "endDate": "2008-12-31" -> NOT PRESENT IN OUR DB
+    # TODO: "active": true -> TO DEFINE ITS MEANING
+    
+    # TODO: "abbreviation": "IB" -> NOT PRESENT IN OUR DB
+    # TODO: "additionalInfo" : { "property1Name" : "property1Value"} -> TO DEFINE ITS MEANING
+    
+    # joins
+    joins_query = 
+    "FROM plant_trials
+    LEFT JOIN plant_scoring_units ON plant_trials.id = plant_scoring_units.plant_trial_id
+    LEFT JOIN plant_accessions ON plant_scoring_units.plant_accession_id = plant_accessions.id
+    LEFT JOIN countries ON plant_trials.country_id = countries.id
+    "
+    
+    total_query = select_query + joins_query + where_query
+    
+    json_wrapping_query = "SELECT row_to_json(row) from ("+total_query+") row"
```

Comment:
Now I understand. I didn't do that because I had some problems with the final coding that I didn't have in this way, but I've just realised that I was doing the json transformation in a wrong way.
I will update all this code.